src/main/java/uk/gov/ons/census/casesvc/model/entity/Case.java

```diff
@@ -115,4 +116,6 @@
 
   @Column(nullable = false, columnDefinition = "BOOLEAN DEFAULT false")
   private boolean undeliveredAsAddressed;
+
+  @Column @UpdateTimestamp private OffsetDateTime lastUpdated;
```

Comment:
minor comment, but because this isn't a `timestamp with time zone` the value returned by the database cannot be guaranteed to be UTC. Strictly speaking, somebody querying our DB from a different time zone might see the incorrect time - it might cause us problems with BST.

Reply:
Added in timestamp with timezone column definition